src/pages/docs/creators/javascript/guides/providers-filesystem.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Golem concepts: filesystem on remote node'
+description: Explore topics related to file system on a VM run on a remote node to better design your custom image.
+pageTitle: VM Filesystem - how the image design impacts my options.
+type: Guide
+---
+
+# Filesystem on VM run on a remote node.
+
+{% alert level="info" %}
+
+This example utilized the `golem/examples-outbound:latest` image deployed on one of the nodes in the `testnet`. Actual numbers will depend on the node on which you run such a task.
+
+{% /alert %}
+
+Your options of what you can do on a remote node will depend on how your image is configured. You already know that transferring data to and from a provider is possible only to/from folders that are configured as `VOLUME` in your image definition. (See [here](/docs/creators/javascript/guides/golem-images) for information about images and how to create one.)
```

Comment:
I would revisit the phrasing a bit "possible only to/from folders that are configured"

To something that states something like "to/from directories thats configured as a VOLUME or a subdirectory of a VOLUME.

Just so that people don't explicit create VOLUMES likes `/golem/work/something/file` but instead just do `/golem` and then they can create whatever from there